feature/gnoi/system/tests/copying_debug_files_test/copying_debug_files_test.go

```diff
@@ -18,20 +18,66 @@ import (
 	"testing"
 	"time"
 
+	"github.com/openconfig/featureprofiles/internal/attrs"
 	"github.com/openconfig/featureprofiles/internal/fptest"
 	hpb "github.com/openconfig/gnoi/healthz"
-	spb "github.com/openconfig/gnoi/system"
+	gnps "github.com/openconfig/gnoi/system"
 	tpb "github.com/openconfig/gnoi/types"
 	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+)
+
+var processName = map[ondatra.Vendor]string{
```

Comment:
pluralize these names: "processNames" and "components"

Reply:
Added changes in latest commit